Rakefile

```diff
@@ -124,8 +124,24 @@ task :git_checks do
   abort red 'Git branch is not up to date please pull' unless branch_up_to_date
 end
 
-task ci: [:erase_devices, :all_checks]
-task all_checks: [:lint, :analyze, :test]
+task ci: [:erase_devices, :lint, :ci_uikit, :ci_swiftui]
+
+task :ci_uikit do
+  task(:all_checks).invoke(EXAMPLE_SCHEME)
+  task(:all_checks).reenable
```

Comment:
need to call `reenable` to avoid Rake not calling the `all_checks` tasks a second time (even if it has different params!!!)